roles/ansible_service_broker/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+ansible_service_broker_image_prefix: ansibleplaybookbundle/ansible-service-broker
+ansible_service_broker_image_tag: latest
+ansible_service_broker_etcd_image_prefix: quay.io/coreos/etcd
```

Comment:
sorry, i may not have been super clear before.  the prefix should exclude `etcd` in this case and then the image would be `{{ prefix }}etcd:{{ tag }}`